docs/administration/admin_panel/corporate_admin_panel.md

```diff
@@ -0,0 +1,13 @@
+---
+description: You can manage Corporate Accounts in the Admin Panel.
```

Comment:
Is this the name we ultimately want as feature name? @juskora @MagdalenaZuba

Reply:
We were urged not to use Corporate Accounts as a name of the feature, instead we can use business accounts/companies profiles/users with customer type. Unfortunately, the UI clearly states Corporate Accounts, so I will rely on @juskora expertise is that left on purpose or by accident.

Reply:
Actually I have no knowledge if it's a leftover or not, so I thin it is safe to use wording described by Magda in a comment earlier.

Reply:
You can also consult Bertrand as he is PM responsible for Customer Portal